torch_xla/csrc/tensor.cpp

```diff
@@ -537,18 +565,18 @@ xla::int64 XLATensor::GetNextTensorId() {
   return id_generator->fetch_add(1);
 }
 
-std::shared_ptr<XLATensor> XLATensor::add(const XLATensor& other,
+std::shared_ptr<XLATensor> XLATensor::add(XLATensor& other,
```

Comment:
Can't we make these `std::shared_ptr` if we need to drop the `const`? This looks weird.

Reply:
I disagree about the mutable alternative being worse, but ok. We can go with this.